fenix/app/src/main/java/org/mozilla/fenix/components/toolbar/DefaultToolbarMenu.kt

```diff
@@ -410,7 +411,7 @@ open class DefaultToolbarMenu(
     val coreMenuItems by lazy {
         val menuItems =
             listOfNotNull(
-                if (shouldUseBottomToolbar) null else menuToolbar,
+                if (shouldUseBottomToolbar || shouldShowMenuToolbar) null else menuToolbar,
```

Comment:
Should this be ```if (!shouldUseBottomToolbar && shouldShowMenuToolbar) menuToolbar else null,`?

Reply:
I was going for `if (shouldUseBottomToolbar || !shouldShowMenuToolbar)` but

Reply:
Sure that works as well.  I just like the positive test cases for easier reading.